scripts/builder/checker_builder/cli.py

```diff
@@ -58,6 +61,10 @@ def generate():
     # Select the price calculation module at runtime based on config
     GENERATE_SRCS[PRICE_SRC] = PRICE_TEMPLATES[checker_config.collateral_type]
 
+    # Generate the token metadata module at runtime based on config
+
+    GENERATE_SRCS[TOKEN_METADATA_SRC] = TOKEN_METADATA_TEMPLATE
```

Comment:
Since the template isn't selected at runtime, I think that we can actually move this into the static `GENERATE_SRCS` on  line 8. The dynamic part which comes from the config will already get passed when the template is rendered below.

Reply:
Ah, right, good point! Done in bd8a9fb :+1: